wdfn-server/waterdata/tests/test_camera.py

```diff
@@ -0,0 +1,94 @@
+from unittest import TestCase
+import sys
+sys.path.append("..")
+import camera
```

Comment:
I think travis isn't happy about this import. Based on the directory structure, what is currently commented out on line 5 may be more appropriate.

Reply:
This was a last resort. I could not get it to import without the legacy path append style. Py would just not find the module one level up in my workspace.
I had `import ..camera` first and then tried `from .. import camera` and others.

Reply:
Were you trying this from PyCharm? I think you need to set up testing in the Preferences (I know because I always have issues, too).